plugins/UsersManager/API.php

```diff
@@ -1448,8 +1448,13 @@ private function getRoleAndCapabilitiesFromAccess($access)
             if ($this->roleProvider->isValidRole($entry)) {
                 $roles[] = $entry;
             } else {
-                $this->checkAccessType($entry);
-                $capabilities[] = $entry;
+                try {
+                    $this->checkAccessType($entry);
+                    $capabilities[] = $entry;
+                } catch (\Exception $e) {
+                    // if capability does not exist any longer (e.g. removed plugin) an exception might be thrown
+                    // we ignore that capability in that case
+                }
```

Comment:
Do we need to check specifically if this is a "capability does not exist" error? Is it possible that other errors might get thrown (if there's a bug, for instance, or if some code in checkAccessType() changes)? Maybe we need an `isValidCapability()` method...

Reply:
changed that 👍